diff --git a/node_modules/@types/vuelidate/vue.d.ts b/node_modules/@types/vuelidate/vue.d.ts
index 64e61c2..6d72e6e 100755
--- a/node_modules/@types/vuelidate/vue.d.ts
+++ b/node_modules/@types/vuelidate/vue.d.ts
@@ -6,7 +6,7 @@ import { Validation } from './vuelidate'
 
 declare module 'vue/types/vue' {
     type ValidationProperties<V> = {
-        [P in Exclude<keyof V, '$v'>]?: Validation & ValidationProperties<V[P]> & ValidationEvaluation
+        [P in Exclude<keyof V, '$v' | '$parent' | '$root' | '$children' | '$options'>]?: Validation & ValidationProperties<V[P]> & ValidationEvaluation
     }
 
     interface ValidationGroups {
